tests/registration/registration.spec.js

```diff
@@ -0,0 +1,291 @@
+import { expect, test } from "@playwright/test";
+
+const name = "aqamay";
+const lastName = "aqarom";
+const email = "aqa-rom7@gmail.com";
+const password = "Password1";
+const repeatPassword = password;
+// empty fields errors
+const errorNameRequiredText = "Name required";
+const errorLastNameRequiredText = "Last name required";
+const errorEmailRequiredText = "Email required";
+const errorPasswordRequiredText = "Password required";
+const errorRepeatPasswordRequiredText = "Re-enter password required";
+// incorrect values
+const incorrectName = "aqamay3";
+const incorrectLastName = "aqarom3";
+const incorrectEmail = "aqa-rom3";
+const incorrectPassword = "password";
+const incorrectRepeatPassword = incorrectPassword;
+// incorrect values errors
+const errorIncorrectNameText = "Name is invalid";
+const errorIncorrectLastNameText = "Last name is invalid";
+const errorIncorrectEmailText = "Email is incorrect";
+const errorIncorrectPasswordText =
+  "Password has to be from 8 to 15 characters long and contain at least one integer, one capital, and one small letter";
+const errorIncorrectRepeatPasswordText = errorIncorrectPasswordText;
+// incorrect length
+const incorrectNameLength = "a";
+const incorrectLastNameLength = "a";
+const incorrectPasswordLength = "Pass1";
+const incorrectRepeatPasswordLength = incorrectPasswordLength;
+// incorrect length errors
+const errorIncorrectNameLengthText =
+  "Name has to be from 2 to 20 characters long";
+const errorIncorrectLastNameLengthText =
+  "Last name has to be from 2 to 20 characters long";
+const errorIncorrectPasswordLengthText =
+  "Password has to be from 8 to 15 characters long and contain at least one integer, one capital, and one small letter";
+const errorIncorrectRepeatPasswordLengthText = errorIncorrectPasswordLengthText;
+// passwords do not match
+const repeatPasswordNotSame = "Password2";
+const errorPasswordLengthNotSameText = "Passwords do not match";
+
+test.describe("Registration", () => {
+  // #1 Register User with correct data
+  test("Register User with correct data", async ({ page }) => {
+    await page.goto("/");
+    const signinButton = page.locator(
+      "button.btn.btn-outline-white.header_signin"
+    );
+    await signinButton.click();
+
+    const popupLogin = page.locator("div.modal-content");
+    await expect(popupLogin, "Log in popup should be visible").toBeVisible();
+
+    const registrationButton = popupLogin.locator(
+      "button.btn.btn-link:nth-child(1)"
+    );
+    await registrationButton.click();
```

Comment:
just click on this button
![image](https://github.com/mayaromanovska/aqa-playwright-1/assets/53534240/79059e76-aaf9-41f2-8e93-6511ea412f76)

Reply:
and move it to `beforeEach` hook